tutorials/katacoda/thanos/2-lts/intro.md

```diff
@@ -1,7 +1,14 @@
-# In progress
+[Thanos](thanos.io) is a set of components that can be composed into a highly available metric system with unlimited storage capacity. It can be added seamlessly on top of existing Prometheus deployments.
 
-🚧 This tutorial is in progress. 🚧
+Thanos allows users to aggregate Prometheus data natively by directly querying the Prometheus API, efficiently compact it, and most importantly, de-duplicate data.
```

Comment:
I feel like this sentence doesn't fit our tutorial as it's about unlimited storage retentation, not connecting Prometheis together

Reply:
Thanks for the review, I'll update it 🤗